jujugui/static/gui/src/app/user/user.js

```diff
@@ -22,6 +22,17 @@ if (typeof this.jujugui === 'undefined') {
   this.jujugui = {};
 }
 
+// XXX j.c.sackett 2016-05-26 Session expiration should be set via config, but
+// as that also requires an update from blues browser we'll do that in a follow
+// up. For now it's set to the same timeout as in blues browser.
+const EXPIRATION_TIME = 24; // Time for a session, in hours
+
+function getExpirationDate() {
```

Comment:
Please add the docstring. Also maybe s/EXPIRATION_TIME/EXPIRATION_HOURS/ and s/getExpirationDate/defaultExpirationDate/ ? [shrug]